docs/migration_guides/opflow_migration.rst

```diff
@@ -0,0 +1,696 @@
+
+1-Qubit Paulis
+~~~~~~~~~~~~~~
+
+.. list-table:: Migration of ``qiskit.opflow.operator_globals`` (1/3)
+   :header-rows: 1
+
+   * - opflow
+     - alternative
+     - notes
+   * - :class:`~qiskit.opflow.X`, :class:`~qiskit.opflow.Y`, :class:`~qiskit.opflow.Z`, :class:`~qiskit.opflow.I`
+     - :class:`~qiskit.quantum_info.Pauli`
+     - For direct compatibility with classes in :mod:`~qiskit.algorithms`, wrap in :class:`~qiskit.quantum_info.SparsePauliOp`.
+   * -
+
+        .. code-block:: python
+
+            from qiskit.opflow import X
+            operator = X ^ X
+
+     -
+
+        .. code-block:: python
+
+            from qiskit.quantum_info import Pauli
+            X = Pauli('X')
+            op = X ^ X
+
+     -
+
+        .. code-block:: python
+
+            from qiskit.quantum_info import Pauli, SparsePauliOp
+            op = Pauli('X') ^ Pauli('X') 
+            
+            # equivalent to:
+            op = SparsePauliOp('XX')
+
+Common non-parametrized gates (Clifford)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. list-table:: Migration of ``qiskit.opflow.operator_globals`` (2/3)
+   :header-rows: 1
+
+   * - opflow
+     - alternative
+     - notes
+
+   * - :class:`~qiskit.opflow.CX`, :class:`~qiskit.opflow.S`, :class:`~qiskit.opflow.H`, :class:`~qiskit.opflow.T`, :class:`~qiskit.opflow.CZ`, :class:`~qiskit.opflow.Swap`
+     - Append corresponding gate to :class:`~qiskit.QuantumCircuit` + :class:`~qiskit.quantum_info.Clifford` + ``.to_operator()``
+     -
+
+   * -
+
+        .. code-block:: python
+
+            from qiskit.opflow import H
+            op = H ^ H
+
+     -
+
+        .. code-block:: python
+
+            from qiskit import QuantumCircuit
+            from qiskit.quantum_info import Clifford
+            qc = QuantumCircuit(2)
+            qc.h(0)
+            qc.h(1)
+            op = Clifford(qc).to_operator()
+
+            # or...
+            qc = QuantumCircuit(1)
+            qc.h(0)
+            H = Clifford(qc).to_operator()
+            op = H ^ H
+
+     -
+
+1-Qubit States
+~~~~~~~~~~~~~~
+.. list-table:: Migration of ``qiskit.opflow.operator_globals`` (3/3)
+   :header-rows: 1
+
+   * - opflow
+     - alternative
+     - notes
+
+   * - :class:`~qiskit.opflow.Zero`, :class:`~qiskit.opflow.One`, :class:`~qiskit.opflow.Plus`, :class:`~qiskit.opflow.Minus`
+     - :class:`~qiskit.quantum_info.Statevector` or :class:`~qiskit.QuantumCircuit` directly
+     -
+
+   * -
+
+        .. code-block:: python
+
+            from qiskit.opflow import Zero, One, Plus, Minus
+
+            state1 = Zero ^ One
+            state2 = Plus ^ Minus
+
+     -
+
+        .. code-block:: python
+
+            from qiskit import QuantumCircuit
+            from qiskit.quantum_info import Statevector
+
+            qc_zero = QuantumCircuit(1)
+            qc_one = copy(qc_zero)
+            qc_one.x(0)
+            state1 = Statevector(qc_zero) ^ Statevector(qc_one)
```

Comment:
StabilizerState is efficient. (It is not exactly the same with opflow, so it depends on the purpose.)